src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -180,7 +180,7 @@
 
 			var $today = $wrapper.find( '.tribe-events-present' ),
 				$mobile_trigger = $wrapper.find( '.mobile-trigger' ),
-				$tribe_grid = $wrapper.find( '#tribe-events-content > .tribe-events-calendar' );
+				$tribe_grid = $wrapper.find( document.querySelector( '#tribe-events-content .tribe-events-calendar' ) );
```

Comment:
I think to make it faster it would be better to do:
````
$wrapper.find( document. getElementById( 'tribe-events-content' ) ).find( '.tribe-events-calendar'  );
```